mithril-client/src/utils/mod.rs

```diff
@@ -0,0 +1,8 @@
+//! Utilities module
+//! This module contains tools needed mostly in services layers.
```

Comment:
Is it still accurate as there are no service in the library?

Reply:
No ! We forgot to overhaul this part of the documentation after the initial copy.